creator-node/src/diskManager.test.js

```diff
@@ -85,23 +85,23 @@ describe('Test DiskManager', function () {
   /**
    * extractCIDsFromFSPath
    */
-  it('Should pass if extractCIDsFromFSPath is passed in a directory and file', function () {
+  it('Should pass if extractCIDsFromFSPath is passed in a directory and file', async function () {
     const path = '/file_storage/files/QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3Grouter/QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3Grinner'
     const matchObj = DiskManager.extractCIDsFromFSPath(path)
     assert.deepStrictEqual(matchObj.isDir, true)
     assert.deepStrictEqual(matchObj.outer, 'QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3Grouter')
     assert.deepStrictEqual(matchObj.inner, 'QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3Grinner')
   })
 
-  it('Should pass if extractCIDsFromFSPath is passed in just a file', function () {
+  it('Should pass if extractCIDsFromFSPath is passed in just a file', async function () {
```

Comment:
same comment here, this is just not async - not sure why all the tests were changed

Reply:
fixed